dll/win32/browseui/brandband.cpp

```diff
@@ -43,7 +43,7 @@ inline void FillSolidRect(HDC dc, const RECT *bounds, COLORREF clr)
     ::ExtTextOut(dc, 0, 0, ETO_OPAQUE, bounds, NULL, 0, NULL);
 }
 
-static const int                            gSmallImageSize = 22;
+//static const int                            gSmallImageSize = 22;
```

Comment:
kept for reference?

Reply:
Yeap, in case the author needs it later on.